pages/dkp/konvoy/2.0/install/air-gapped/new/index.md

```diff
@@ -71,13 +71,13 @@ enterprise: false
     kubectl get clusters,awsclusters,machinepools,awsmachinepools
     ```
 
-1.  Wait for the cluster control plane to be ready:
+1.  Wait for the cluster control-plane to be ready:
```

Comment:
Just a general question - this is different in diff places in the docs (either `control plane` or `control-plane`) -- what is the preference on this?
@cmastr @samvantran @dkoshkin

Reply:
Is this supposed to be a "control-plane node"? Like, is this an adjective or is "control plane" a noun? That makes a difference in the dash.

Reply:
yeah that's the shortened version should be `control-plane node`